docs_user/modules/assembly_ospdo-scale-down-pre-database-adoption.adoc

```diff
@@ -0,0 +1,215 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  labels:
+    osp/interface: enp7s0
+  name: <ostest-master-node>
+spec:
+  desiredState:
+    dns-resolver:
+      config:
+        search: []
+        server:
+        - 172.22.0.1
+    interfaces:
+    - description: internalapi vlan interface
+      name: enp7s0.20
+      state: up
+      type: vlan
+      vlan:
+        base-iface: enp7s0
+        id: 20
+        reorder-headers: true
+      ipv4:
+        address:
+        - ip: 172.17.0.7
+          prefix-length: 24
+        enabled: true
+        dhcp: false
+      ipv6:
+        enabled: false
+    - description: storage vlan interface
+      name: enp7s0.30
+      state: up
+      type: vlan
+      vlan:
+        base-iface: enp7s0
+        id: 30
+        reorder-headers: true
+      ipv4:
+        address:
+        - ip: 172.18.0.7
+          prefix-length: 24
+        enabled: true
+        dhcp: false
+      ipv6:
+        enabled: false
+    - description: storagemgmt vlan interface
+      name: enp7s0.40
+      state: up
+      type: vlan
+      vlan:
+        base-iface: enp7s0
+        id: 40
+        reorder-headers: true
+      ipv4:
+        address:
+        - ip: 172.19.0.7
+          prefix-length: 24
+        enabled: true
+        dhcp: false
+      ipv6:
+        enabled: false
+    - description: tenant vlan interface
+      name: enp7s0.50
+      state: up
+      type: vlan
+      vlan:
+        base-iface: enp7s0
+        id: 50
+        reorder-headers: true
+      ipv4:
+        address:
+        - ip: 172.20.0.7
+          prefix-length: 24
+        enabled: true
+        dhcp: false
+      ipv6:
+        enabled: false
+    - description: Configuring Bridge br-ctlplane with interface enp7s0
+      name: br-ctlplane
+      mtu: 1500
+      type: linux-bridge
+      state: up
+      bridge:
+        options:
+          stp:
+            enabled: false
+        port:
+          - name: enp1s0
+            vlan: {}
+      ipv4:
+        address:
+        - ip: 172.22.0.53
+          prefix-length: 24
+        enabled: true
+        dhcp: false
+      ipv6:
+        enabled: false
+    - bridge:
+        options:
+          stp:
+            enabled: false
+        port:
+        - name: enp6s0
+      description: Linux bridge with enp6s0 as a port
+      ipv4:
+        enabled: false
+      ipv6:
+        enabled: false
+      mtu: 1500
+      name: br-external
+      state: up
+      type: linux-bridge
+  nodeSelector:
+    kubernetes.io/hostname: <ostest-master-node>
+    node-role.kubernetes.io/worker: ""
+EOF
+
+oc apply -f /tmp/node3_nncp.yaml
+----
++
+. Get remaining OSPdO operands and delete everything, except DO NOT DELETE OpenStackBaremetalSets nor OpenStackProvisionServers yet
++
+----
+for i in $(oc get crd | grep osp-director | grep -v baremetalset | grep -v provisionserver | awk {'print $1'}); do echo Deleting $i...; oc delete $i -n openstack --all; done
+----
++
```

Comment:
```suggestion
. Get the remaining OSPdO operands and delete everything. Do no delete `OpenStackBaremetalSets` and `OpenStackProvisionServers` :
+
----
$ for i in $(oc get crd | grep osp-director | grep -v baremetalset | grep -v provisionserver | awk {'print $1'}); do echo Deleting $i...; oc delete $i -n openstack --all; done
----

```
@pinikomarov Does "everything" refer to "operands"? Or "services"? I'd like to use a more specific word.

Reply:
ok , 
and I'll change to "resources" , that's more specific